lightning/src/ln/msgs.rs

```diff
@@ -442,6 +442,13 @@ pub enum NetAddress {
 		/// The port on which the node is listening
 		port: u16,
 	},
+	/// A hostname/port on which the peer is listening.
+	Hostname {
+		/// The hostname on which the port is listening.
```

Comment:
Think "port" should be "node" or "peer"?

```suggestion
		/// The hostname on which the node is listening.
```

Reply:
Done.